shared/test/test_animation_library_api.rb

```diff
@@ -44,6 +44,21 @@ def test_get_gamelab_animation
     AWS::S3.s3 = nil
   end
 
+  def test_get_spritelab_manifest
+    contents = 'TEST_MANIFEST'
+    # Ensure the shared S3 client is used by stubbing the client with the expected response.
+    AWS::S3.s3 = Aws::S3::Client.new(
+      stub_responses: {
+        get_object: {body: contents, content_type: 'json'}
+      }
+    )
+    get '/api/v1/animation-library/animation-manifests/manifests/test_manifests.png'
```

Comment:
i'm not sure i understand what this test is testing -- can you explain?

Reply:
Yup, this is my bad, I was using the wrong path here. I meant to test: `get '/api/v1/animation-library/manifest/spritelab/en_us'` - will update that in the next PR. As far as the purpose of the test (with the correct path), this is mostly because as I was editing the api to use the new path, I realized there was no test coverage of the middleware function, so I wanted to add a simple test here as a confidence-check. Whether it is useful or not, I'm not sure, so feel free to let me know if you think it is unnecessary.

Reply:
ah, okay -- that makes sense, and i do think that's a valuable test and we should make that change as part of this PR